inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -186,30 +187,3 @@ def date_validator(form, field):
                 break
         else:
             raise StopValidation(message)
-
-
-def _is_pdf_link(url):
```

Comment:
Moving this to the more central utils, since I am going to use it also for workflows.